README.md

```diff
@@ -89,74 +37,123 @@ And an `.env` file with following content:
 
-##### Normal start
+### Running on mac silicon
 
-This should be your go-to way of starting the stack.
+Running the application on mac silicon can cause some troubles. For this reason an extra docker-compose file has been included, this is the file docker-compose.mac.yml, this file should be included when starting the stack. The command `docker-compose up -f docker-compose.yml -f docker-compose.dev.yml up -d` now becomes `docker compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.mac.yml up -d`
+There are two main painpoints:
+
+1. Mac has an arm64 processor, a lot of the services don't have a multi-platform image. In the case they only have a amd64 image, docker will gave you a warning about this. In general this is not a real problem since your macbook can just emulate amd64, but still the warnings are annoying, so these are suppressed.
+2. At the moment this project was setup the service mu-identifier and mu-authorization weren't working for mac (at least on my device), so you have to build these yourself, and gave them the appropriate image name and tag.
+
+### Upgrading your setup
+
+Once installed, you may desire to upgrade your current setup to follow development of the main stack. The following example describes how to do this.
+
+For the dev setup, we assume you'll pull more often and thus will most likely clear the database separately:
 
 ```
-docker-compose up # or 'docker-compose up -d' if you want to run it in the background
+# This assumes the .env file has been set. Cf. supra in the README.md
+# Bring the application down
+docker compose down
+# Pull in the changes
+git pull origin master
+# Launch the stack
+docker compose up
 ```
 
-Always double check the status of the migrations `docker-compose logs -f --tail=100 migrations`
-Wait for everything to boot to ensure clean caches.
+As with the initial setup, we wait for everything to boot to ensure clean caches. You may choose to monitor the migrations service in a separate terminal to and wait for the overview of all migrations to appear: `docker compose logs -f --tail=100 migrations`.
 
-Probably the first thing you'll want to do, is see wether the app is running correctly. The fastest way forward is creating a `docker-compose.override.yml` file next to the other `docker-compose.yml` files, and add
+Once the migrations have ran, you can go on with your current setup.
+
+### Cleaning the database
+
+At some times you may want to clean the database and make sure it's in a pristine state, it is always a good idea to backup your data first.
 
 ```
-# (...)
-  loket:
-    ports:
-      - 4205:80
+# This assumes the .env file has been set. Cf. supra in the README.md
+# Bring down our current setup
+docker compose down
+# Back-up your database folder
+mv data/db data/db-bak
```

Comment:
it's worthwhile adding something about making a checkpoint here.
i.e. go to the conductor interface on localhost:8890, click isql on the top left and run the query `CHECKPOINT;`
Else you may lose data that wasn't written to disk yet